requirements/doc.txt

```diff
@@ -1,6 +1,7 @@
 # Generated via tools/generate_requirements.py.
 # Do not edit this file; modify `pyproject.toml` instead and run `python tools/generate_requirements.py`.
 sphinx>=5.0.0
+intersphinx_registry
```

Comment:
Please add this by running `python tools/generate_requirements.py`

Reply:
Never mind, you got lucky : )

Reply:
also why is not this file full content not just

```
.[doc]
```

That woudl aleviate having to regenerate it.

Reply:
Yeah some people complained to have these individual files 😅 I don't recall their exact justification for doing it like that.

Reply:
Maybe like for other project, those files where the source of truth at the beginning and are not generated ? 

the `.[foo]` syntax in requirements.txt is also not widely known.